operations.md

```diff
@@ -1,7 +1,8 @@
-# Operations Manual: logs and alarms
+# Operations Manual: logs, alarms and stats
 ## Index
 
 * [Logs](#logs)
+* [Cache Stats](#stats)
```

Comment:
With the new operation for reseting the cache at all, maybe it's better to name it this way:

```suggestion
* [Cache management](#cache)
```